components/fieldgroups/experience-event/experienceevent-campaign-delivery-log.schema.json

```diff
@@ -0,0 +1,138 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/experienceevent-campaign-delivery-log",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Campaign Delivery Logs",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Campaign Delivery Events such as sent, bounces, failures.",
+  "definitions": {
+    "experienceevent-campaignlog": {
+      "properties": {
+        "xdm:directMarketing": {
+          "type": "object",
+          "description": "Captures direct marketing related details.",
+          "properties": {
+            "xdm:externalId": {
+              "title": "External Identifier",
+              "type": "string",
+              "description": "External Identifier that defines the primary identity value for profile and audience."
+            },
+            "xdm:campaignId": {
+              "title": "Campaign Identifier",
+              "type": "string",
+              "description": "Identifier of the marketing campaign to which activity originating this message belongs to."
+            },
+            "xdm:campaignName": {
+              "title": "Campaign Name",
+              "type": "string",
+              "description": "Internal Name that uniquely identifies the Campaign."
+            },
+            "xdm:campaignLabel": {
+              "title": "Campaign Label",
+              "type": "string",
+              "description": "A human-friendly label of the marketing campaign which is originating this message."
+            },
+            "xdm:broadlogId": {
+              "title": "Delivery Log Identifier",
+              "type": "string",
+              "description": "Identifier of the marketing campaign delivery log this event belongs to."
+            },
+            "xdm:deliveryId": {
+              "title": "Delivery Identifier",
+              "type": "string",
+              "description": "The delivery originating this message."
+            },
+            "xdm:visitorId": {
```

Comment:
@jishah-adobe  - What is the different between a visitorId and a recipientId? When would they be different?

Reply:
Visitor is another form of audience that links visitors to the services that they have subscribed on social platforms (mostly twitter and facebook).  Campaign maintains different event logs for each audience type (recipient, visitor, services, real and batch transactional events, exclusions and so on).  Each of them links to a different look up tables and basically we put the primary identifiers here in the schema